frontend/src/views/GameView.vue

```diff
@@ -64,6 +90,21 @@ function abandon() {
     router.push({ name: 'rooms' })
 }
 
+function drawRquest() {
+    DrawRequestAction(game.repository)
+}
+
+function acceptDraw() {
+    DrawResponseAction(game.repository, true)
+    game.setEndGameReason('draw')
```

Comment:
`'draw'` should be a constant variable instead of a "magic string" copy-pasted anywhere needed, since this copy-paste strategy may result in typos and errors difficult to find.